### PR TITLE
Update and rename column_naming_convention.md to column_naming_and_or…

### PR DESCRIPTION
**Pull request by AWS-ZachErdman**
Updated attribute to be more clear this includes order rules too.

### CODE REVIEW

**Review by cnharris10 — APPROVED** [review]

**Comment by jpradocueva** [comment]
Approved by the group on April 18

File: specification/attributes/column_naming_and_ordering_convention.md [hunk 1]
```diff
@@ -1,23 +1,24 @@
-# Column Naming Convention
+# Column Naming and Ordering Convention
```
Comment (flanakin) [inline specification/attributes/column_naming_and_ordering_convention.md:1]: nit: Should we remove "Convention"? "Naming convention" makes sense, but "ordering convention" doesn't seem right. Given it's an attribute, it feels unnecessary. I'm fine either way. Just a nit.
```suggestion
# Column Naming and Ordering
```

File: specification/attributes/column_naming_and_ordering_convention.md [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
-Column IDs provided in cost data following a consistent naming convention reduce friction for FinOps practitioners who consume the data for analysis, reporting, and other use cases.
+Column IDs provided in cost data following a consistent naming and ordering convention reduce friction for FinOps practitioners who consume the data for analysis, reporting, and other use cases.
 
-All columns defined in the [FOCUS](#glossary:finops-cost-and-usage-specification) specification MUST follow the naming requirements listed below.
+All columns defined in the [FOCUS](#glossary:finops-cost-and-usage-specification) specification MUST follow the naming and ordering requirements listed below.
 
 ## Attribute ID
 
-ColumnNamingConvention
+ColumnNamingAndOrderingConvention
```
Comment (flanakin) [inline specification/attributes/column_naming_and_ordering_convention.md:9]:
```suggestion
ColumnNamingAndOrdering
```

File: specification/attributes/column_naming_and_ordering_convention.md [hunk 3]
```diff
@@ -1,23 +1,24 @@
...
 
 ## Attribute Name
 
-Column Naming Convention
+Column Naming and Ordering Convention
```
Comment (flanakin) [inline specification/attributes/column_naming_and_ordering_convention.md:13]:
```suggestion
Column Naming and Ordering
```

**Review by flanakin — APPROVED** [review]

**Review by mike-finopsorg — APPROVED** [review]

File: specification/attributes/column_naming_and_ordering_convention.md [hunk 4]
```diff
@@ -1,23 +1,24 @@
-# Column Naming Convention
+# Column Naming and Ordering
```
Comment (flanakin) [inline specification/attributes/column_naming_and_ordering_convention.md:1]: Can you also update the changelog in the root to add this rename as a subbullet under Changed > Column naming convention?

**Review by udam-f2 — APPROVED** [review]

**Comment by AWS-ZachErdman** [comment]
Replacement PR created from a branch directly in the repo that avoids conflicts: https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/416

**Comment by jpradocueva** [comment]
This PR has been superseded by PR #416 due to some editing conflicts.

**Pull request closed without merging**